a11ym

```diff
@@ -190,19 +188,34 @@ urls.drain = function () {
     }
 };
 
-var crawler             = new Crawler(url.hostname);
-crawler.initialPath     = url.path;
-crawler.initialPort     = url.port || 80;
-crawler.initialProtocol = (url.protocol || 'http').replace(':', '');
-crawler.interval        = 100;
-crawler.maxConcurrency  = 5;
-crawler.maxDepth        = +program.maximumDepth;
+var parseURL = function (url) {
+    var parsed = URL.parse(url);
+
+    if (!parsed.protocol) {
+        parsed.protocol = 'http';
+    } else {
+        parsed.protocol = parsed.protocol.replace(':', '');
+    }
+
+    if (!parsed.port) {
+        parsed.port = 'http' === parsed.protocol ? 80 : 443;
```

Comment:
This will not work on our client project as the crawler explicitely add the port the the URL when something is set and our Router does not recognize an URL with a port in it.

I mostly agree that it is our Router fault, but still, we force the addition of a port to an URL which otherwise has none, not sure it is a great idea.

IHMO, we should not change the value of this.